### PR TITLE
Add importer support for dict keys

### PR DESCRIPTION
**Pull request by stphivos**
e.g. `host: '^settings.statsd_config["host"]'`

### CODE REVIEW

File: pyrovider/meta/ioc.py [hunk 1]
```diff
@@ -10,5 +11,12 @@ def get_obj(class_path: str) -> type:
         module_parts = class_path.split('.')
         module_name = ".".join(module_parts[:-1])
         module = importlib.import_module(module_name)
+        obj = module_parts[-1:][0]
 
-        return module.__dict__[module_parts[-1:][0]]
+        matches = re.findall(r'\["[\w-]+"]', obj)
```
Comment (sebastiandev) [inline pyrovider/meta/ioc.py:16]: what is this for? I'm having troubling understanding the logic

Comment (sebastiandev) [reply pyrovider/meta/ioc.py:16]: oh. is this to support dictionary indexing as part of the parameter, when referencing a config dict?

Comment (sebastiandev) [reply pyrovider/meta/ioc.py:16]: Not sure if this is worth it. Wouldn't it be better to just convert the settings dict into a DictObject and use regular dot notation?

Comment (stphivos) [reply pyrovider/meta/ioc.py:16]: yeah good idea. i can try that thx!

**Pull request closed without merging**